tests/testthat/test-garnish_variants_intersection.R

```diff
@@ -14,10 +14,18 @@ testthat::test_that("garnish_variants intersection", {
       utils::download.file(paste0("http://get.rech.io/", .), .)
 
   # run test
-    dto <- garnish_variants(c("antigen.garnish_example_mutect2.vcf",
+    dto_i <- garnish_variants(c("antigen.garnish_example_mutect2.vcf",
                      "antigen.garnish_example_strelka.vcf"))
 
-  testthat::expect_equal(dto %>% length, 21)
-  testthat::expect_equal(dto %>% nrow, 4)
+  testthat::expect_equal(dto_i %>% length, 23)
+  testthat::expect_equal(dto_i %>% nrow, 64)
+
+    # run test
+    dto_ni <- garnish_variants(c("antigen.garnish_example_mutect2.vcf",
+                     "antigen.garnish_example_strelka.vcf"),
+            intersect = FALSE)
+
+  testthat::expect_equal(dto_ni %>% length, 23)
+  testthat::expect_equal(dto_ni %>% nrow, 118)
```

Comment:
I think ideally changes to union_vcf above will not affect this test, just the content of the table.  this table likely had NA's for REF, CHROM, POS etc. right?

Reply:
Yes you are right. We actually want to subset to variants present in both and merge those, then `rbindlist` to variants present in either vcf.

Reply:
Fixed